README.md

```diff
@@ -135,48 +173,65 @@ print("Probability with `predict_proba`: ", y_proba_fhe)
 ```
 
-This example is explained in more detail in the [linear model documentation](docs/built-in-models/linear.md). Concrete ML built-in models
-have APIs that are almost identical to their scikit-learn counterparts. It is also possible to convert PyTorch networks to FHE with the Concrete ML conversion APIs. Please refer to the [linear models](docs/built-in-models/linear.md), [tree-based models](docs/built-in-models/tree.md) and [neural networks](docs/built-in-models/neural-networks.md) documentation for more examples, showing the scikit-learn-like API of the built-in
-models.
-
-## Documentation.
+*This example is explained in more detail in the [linear model documentation](https://docs.zama.ai/concrete-ml/built-in-models/linear).*
 
-Full, comprehensive documentation is available here: [https://docs.zama.ai/concrete-ml](https://docs.zama.ai/concrete-ml).
+Concrete ML built-in models have APIs that are almost identical to their scikit-learn counterparts. It is also possible to convert PyTorch networks to FHE with the Concrete ML conversion APIs. Please refer to the [linear models](docs/built-in-models/linear.md), [tree-based models](docs/built-in-models/tree.md) and [neural networks](docs/built-in-models/neural-networks.md) documentation for more examples, showing the scikit-learn-like API of the built-in models.
 
-## Online demos and tutorials.
-
-Various tutorials are given for [built-in models](docs/built-in-models/ml_examples.md) and for [deep learning](docs/deep-learning/examples.md) In addition, several complete use-cases are explored:
+<p align="right">
+  <a href="#table-of-contents" > ↑ Back to top </a> 
+</p>
 
-- [Encrypted Large Language Model](use_case_examples/llm/): convert a user-defined part of a Large Language Model for encrypted text generation. Shows the trade-off between quantization and accuracy for text generation and shows how to run the model in FHE.
+## Resources 
 
-- [Credit Scoring](use_case_examples/credit_scoring/): predict the chance of a given loan applicant defaulting on loan repayment while keeping the user's data private. Shows how Concrete ML models easily replace their scikit-learn equivalents
+### 🟨 Demos 
 
-- [Health diagnosis](use_case_examples/disease_prediction/): based on a patient's symptoms, history and other health factors, give
-  a diagnosis using FHE to preserve the privacy of the patient.
+#### Live demos on Hugging Face
 
-- [Private inference for federated learned models](use_case_examples/federated_learning/): private training of a Logistic Regression model and then import the model into Concrete ML and perform encrypted prediction
+- [Credit Scoring](https://huggingface.co/spaces/zama-fhe/credit_card_approval_prediction):  predicting the chance of a given loan applicant defaulting on loan repayment while keeping the user's data private. This demo shows how Concrete ML models easily replace their scikit-learn equivalents.
+  - [code](use_case_examples/credit_scoring/)
```

Comment:
this is not exact : the code for the HF space (https://huggingface.co/spaces/zama-fhe/credit_card_approval_prediction) is not `use_case_examples/credit_scoring/` , that latter is a different use case that also talks about Credit Scoring 😅  

it would be more accurate to suggest https://huggingface.co/spaces/zama-fhe/credit_card_approval_prediction/tree/main as the code for the HF space on credit scoring

Reply:
ah yes you're right @yuxizama , the description should be updated ! Maybe something like : 
> Predicting credit scoring card approval application in which sensitive data can be shared and analyzed without exposing the actual information to neither the three parties involved nor the server processing it. 

(slightly adapted from the associated linkedin post)